apps/framework-docs/src/pages/moose/reference/ts-moose-lib.mdx

```diff
@@ -40,6 +40,114 @@ interface ApiUtil {
 }
 ```
 
+## Date and Time Types
+
+Moose provides type aliases for temporal data that map to ClickHouse's DateTime types, 
+giving you a choice between Date objects and strings based on your precision requirements.
+
+### `DateTime`
+A type alias for JavaScript's native `Date` object, suitable for datetime values with up to millisecond precision.
+```ts
+interface Event {
+  // Stored as ClickHouse DateTime (second precision)
```

Comment:
I got confused by this. It says second precision, but above & below in this same section says millisecond precision.